spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -71,119 +71,119 @@
+  it 'does not register an offense for ACRONYMClassNames' do
     expect_no_offenses(<<-RUBY, 'abc_one/two_spec.rb')
       describe ABCOne::Two do; end
     RUBY
   end
 
-  it 'handles ALLCAPS class names' do
+  it 'does not register an offense for ALLCAPS class names' do
     expect_no_offenses(<<-RUBY, 'allcaps_spec.rb')
       describe ALLCAPS do; end
     RUBY
   end
 
-  it 'handles alphanumeric class names' do
+  it 'does not register an offense for alphanumeric class names' do
     expect_no_offenses(<<-RUBY, 'ipv4_and_ipv6_spec.rb')
       describe IPV4AndIPV6 do; end
     RUBY
   end
 
-  it 'checks instance methods' do
+  it 'does not register an offense for instance methods' do
     expect_no_offenses(<<-RUBY, 'some/class/inst_spec.rb')
       describe Some::Class, '#inst' do; end
     RUBY
   end
 
-  it 'checks class methods' do
+  it 'does not register an offense for class methods' do
     expect_no_offenses(<<-RUBY, 'some/class/inst_spec.rb')
       describe Some::Class, '.inst' do; end
     RUBY
   end
 
-  it 'allows flat hierarchies for instance methods' do
+  it 'does not register an offense for flat hierarchies for instance methods' do
     expect_no_offenses(<<-RUBY, 'some/class_inst_spec.rb')
       describe Some::Class, '#inst' do; end
     RUBY
   end
 
-  it 'allows flat hierarchies for class methods' do
+  it 'does not register an offense for flat hierarchies for class methods' do
     expect_no_offenses(<<-RUBY, 'some/class_inst_spec.rb')
       describe Some::Class, '.inst' do; end
     RUBY
   end
 
-  it 'allows subdirs for instance methods' do
+  it 'does not register an offense for subdirs for instance methods' do
     filename = 'some/class/instance_methods/inst_spec.rb'
     expect_no_offenses(<<-RUBY, filename)
       describe Some::Class, '#inst' do; end
     RUBY
   end
 
-  it 'allows subdirs for class methods' do
+  it 'does not register an offense for subdirs for class methods' do
     filename = 'some/class/class_methods/inst_spec.rb'
     expect_no_offenses(<<-RUBY, filename)
       describe Some::Class, '.inst' do; end
     RUBY
   end
 
-  it 'ignores non-alphanumeric characters' do
+  it 'does not register an offense for non-alphanumeric characters' do
     expect_no_offenses(<<-RUBY, 'some/class/pred_spec.rb')
       describe Some::Class, '#pred?' do; end
     RUBY
   end
 
-  it 'allows bang method' do
+  it 'does not register an offense for bang method' do
     expect_no_offenses(<<-RUBY, 'some/class/bang_spec.rb')
       describe Some::Class, '#bang!' do; end
     RUBY
   end
 
-  it 'allows flexibility with predicates' do
+  it 'does not register an offense for flexibility with predicates' do
     filename = 'some/class/thing_predicate_spec.rb'
     expect_no_offenses(<<-RUBY, filename)
       describe Some::Class, '#thing?' do; end
     RUBY
   end
 
-  it 'allows flexibility with operators' do
+  it 'does not register an offense for flexibility with operators' do
```

Comment:
Those two do not read right.
In any case, it allows flexibility not only for operators/predicates:
```
            name_pattern(method_name),
            '[^/]*_spec\.rb'
```
but actually for any spec:
```ruby
  it 'allows flexibility with predicates' do
    filename = 'some/class/thing_predicate_spec.rb'
    expect_no_offenses(<<-RUBY, filename)
      describe Some::Class, '#thing' do; end
    RUBY
  end
```
passes just fine.

I'd suggest changing the first example to "does not register an offence for an arbitrary spec suffix".

And the second to "does not register an offence for an arbitrary spec name for an operator method".